Makefile

```diff
@@ -14,6 +14,9 @@ BUILD_DATE:=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 ALL_ARCHITECTURES=amd64 arm arm64 ppc64le s390x
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
+BINARIES_ARCHITECTURES=amd64 arm64
```

Comment:
To be consistent with the image that we build and publish, I think it would be better to reuse `ALL_ARCHITECTURES` instead